tools/jsrun.py

```diff
@@ -33,7 +33,13 @@ def make_command(filename, engine=None, args=[]):
   # label a path to nodejs containing a 'd8' as spidermonkey instead.
   jsengine = os.path.split(engine[0])[-1]
   # Use "'d8' in" because the name can vary, e.g. d8_g, d8, etc.
-  return engine + [filename] + (['--expose-wasm', '--'] if 'd8' in jsengine or 'jsc' in jsengine else []) + args
+  is_d8 = 'd8' in jsengine
```

Comment:
I was thinking more "these are engine-specific modifications, so slice it by engine". If we start actually growing more engine-specific flags we can squish it around later tho.